internal/pkg/vm/actor/builtin/account/account_encoding_gen.go

```diff
@@ -1,20 +1,19 @@
 package account
-	
-/* 
+
+/*
```

Comment:
Looks like this got linted. It's unfortunate that the generated code isn't linter compatible. This could cause churn, but it's probably more likely to get linted multiple times before it gets regenerated, so probably best to leave this the way it is.